cmake/CMakeLists.txt

```diff
@@ -132,8 +132,8 @@ add_definitions(-DEIGEN_MPL2_ONLY)
 
 if(onnxruntime_CROSS_COMPILING)
   set(CMAKE_CROSSCOMPILING ON)
-  string(APPEND CMAKE_CXX_FLAGS " -Wno-error=attributes")
-  string(APPEND CMAKE_C_FLAGS " -Wno-error=attributes")
+  #string(APPEND CMAKE_CXX_FLAGS " -Wno-error=attributes")
```

Comment:
Wondering if this will break @RandySheriff 's arm32 fix?

Reply:
Updated to work for --arm as well as --arm64. We need another option to specify bit-ness.